rs/nns/governance/canister/governance.did

```diff
@@ -577,6 +626,62 @@ type Neuron = record {
   transfer : opt NeuronStakeTransfer;
   known_neuron_data : opt KnownNeuronData;
   spawn_at_timestamp_seconds : opt nat64;
+
+  // When a neuron is first created, this is set to create_timestamp_seconds.
+  //
+  // This is part of the "periodic confirmation" feature, which was proposed and
+  // (overwhelmingly) adopted in NNS motion proposal 132411:
+  // https://dashboard.internetcomputer.org/proposal/132411
+  //
+  // This gets updated to the current time when the user performs related
+  // actions, as described in the aforementioned proposal.
```

Comment:
```suggestion
  // This gets updated to the current time when the user performs related
  // actions: voting, set following, or confirm following.
```
I think it is preferrable if this is self-contained and one does not need to refer to an external proposal to learn which actions reset this

Reply:
I agree with your changes (but I think we need to emphasize that only DIRECT voting counts, not indirect voting.).

This still does not describe the **actual consequences** of not updating, but the consequences are the most interesting piece of information. We could either include that inline, or link out. To avoid repetition, I think linking out is best.

Reply:
Good point, let's use "vote directly" .
OK, I'll leave the last part to you, I was thikning maybe the API doesn't need to explain all the consequences of the field but maybe you are right that it is better

Reply:
I think there is a middle ground here: at least describe **abstractly** that there are negative consequences, but defer to other sources for a more **precise** description of those negative consequences.

Reply:
Done.